### DISCUSSION TITLE
What's the best way to run in development?

### DISCUSSION THREAD

**Discussion by NathanBeddoeWebDev** [harlan-zw/unlighthouse#91] [General]
Hello!

I've forked the project to take a look at some weirdness with overflows, and some other sorting issues. If I were making frontend changes, at the moment I'm just rebuilding the whole project and running the cli against windicss because pnpm dev doesn't seem to work (something about client:watch not existing). Surely there's a better way to do this?

**Comment by NathanBeddoeWebDev** [comment]
I've just found https://github.com/antfu/contribute, will take a peek at that and see what I can do.

**Comment by tuminzee** [comment]
> Hello!
> 
> I've forked the project to take a look at some weirdness with overflows, and some other sorting issues. If I were making frontend changes, at the moment I'm just rebuilding the whole project and running the cli against windicss because pnpm dev doesn't seem to work (something about client:watch not existing). Surely there's a better way to do this?

I think you have found a solution for this already, but I am happy to help in any ways, I was able to setup, along with starting a debug mode
